### PR TITLE
Calls

### PR DESCRIPTION
**Pull request by futuregov**
Closes https://trello.com/c/kvRhf5LN/155-changes-to-schedule-call-screen and https://trello.com/c/MlOtjTek/156-changes-to-log-call-screen

@em-cd I am going to do an interactive rebase to tidy up the commit messages, but the code should be good for review.

### CODE REVIEW

File: app/controllers/assessments_controller.rb [hunk 1]
```diff
@@ -159,8 +159,12 @@ def find_contact_id; end
 
   def log_assessment
     @need = Need.new(assessment_params.merge(contact_id: @contact.id, start_on: Date.today))
+    @need.name = "#{@need.category} call - #{Time.zone.now.strftime('%d/%m/%y')}"
```
Comment (kittysquee) [inline app/controllers/assessments_controller.rb:162]: Should this follow the same date formatting that we use for contacts? `%d/%m/%Y`?

File: db/schema.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_01_06_132742) do
+ActiveRecord::Schema.define(version: 2021_05_20_091922) do
```
Comment (em-cd) [inline db/schema.rb:13]: 😄

**Review by em-cd — COMMENTED** [review]
Looks good, well done for wrangling these changes! 🙌 

There's a minor visual tweak which you could try to tackle if you have time, which is to make the note select box wider. I haven't used select2 before so I'm not sure how difficult that would be, but there's definitely space for it and it would help to be able to read the whole category once you've selected it:

Screenshot 2021-05-26 at 11 03 06

File: app/helpers/notes_helper.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module NotesHelper
+  def notes_category_helper(category)
+    categories = {
```
[the remark this answers is not in the document]
Comment (tosbourn) [reply app/helpers/notes_helper.rb:5]: That is a great shout, will add that now. I did write it in a bit of a "oh come on now cucumber" but will definitely write a more grown up rationale haha

**Review by kittysquee — APPROVED** [review]

**Review by em-cd — APPROVED** [review]
Perfect, thanks!

**Pull request merged**